commands/command_utils.py

```diff
@@ -48,7 +49,8 @@ def detect_sap_hana():
     if os.path.exists(HANA_BASE_PATH):
         for entry in os.listdir(HANA_BASE_PATH):
             # Does /hana/shared/{entry}/exe/linuxx86_64/hdb/sapcontrol exist?
-            if os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH)):
+            if (os.path.exists(os.path.join(HANA_BASE_PATH, entry, HANA_SAPCONTROL_PATH_X86_64))
```

Comment:
Imho 
`if any(os.path.exists(os.path.join(HANA_BASE_PATH, entry, sapcontrol_path)) for sapcontrol_path in [HANA_SAPCONTROL_PATH_X86_64, HANA_SAPCONTROL_PATH_PPC64LE])`
is a bit more readable

Reply:
@pirat89 Can be :) I'd leave it to Irina which option to choose

Reply:
@pirat89 I've checked on your branch, it looks neat. Do you want to close this PR, and open one with your 'fix'?

Reply:
@pirat89 if you like it, just merge to you branch & squash it :)